pkg/client/devops/jenkins/pure_request.go

```diff
@@ -48,12 +48,26 @@ func (j *Jenkins) SendPureRequestWithHeaderResp(path string, httpParameters *dev
 	client := &http.Client{Timeout: 30 * time.Second}
 
 	header := httpParameters.Header
+	if header == nil {
+		header = http.Header{}
```

Comment:
I don't think this code line can work. After this line, the variable `header` will be a new one.

Reply:
If header is nil, it should be a new one(map[string]string).

Reply:
Yes, but it would update to `httpParameters.Header`

Reply:
In fact, the header should be cloned from `httpParameters.Header`. I will fix it later.